boards/riscv/hifive1/board.cmake

```diff
@@ -1,9 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
-set(EMU_PLATFORM qemu)
+set(EMU_PLATFORM renode qemu)
+set(EMU_PLATFORM_DEFAULT renode)
```

Comment:
let's avoid the `EMU_PLATFORM_DEFAULT` variable.
If `EMU_PLATFORM` is a list, then the first entry in that list should be considered the default emulation platform.

Reply:
Removed the `EMU_PLATFORM_DEFAULT` variable.